text/0027-manual-disabling-of-flaky-tests.md

```diff
@@ -28,25 +28,23 @@ https://sentry.io/organizations/sentry/dashboard/7997/?project=5350637&project=4
 **a test should be disabled if it has flaked on the primary branch**
 
 - identify the owner of the test in question
-    - often `.github/CODEOWNERS` can help
+    - often [.github/CODEOWNERS](https://github.com/getsentry/sentry/blob/master/.github/CODEOWNERS) can help
     - otherwise utilize `git log -- path/to/the/test/file`
 - find the corresponding sentry issue:
     - for example: [django tests](https://sentry.io/organizations/sentry/issues/?limit=5&project=2423079&query=&sort=freq&statsPeriod=14d)
 - create a JIRA ticket (or if the team uses github issues, do that instead)
     - subject: `disabled flaky test: <testname>`
     - body: (something like this)
 
-      ```
-          the following test is flaky so it is being disabled:
+```
+The following test is flaky so it is being disabled: <name of tests>
 
-          here is the sentry issue: <<< link here >>>
+Here is the sentry issue: <<< link here >>>
 
-          ```
-          <<< test failure output here >>>
-          ```
```

Comment:
I think a different trick that can be used here is to alternate backticks and ~ for the code fences -- I believe github supports both so it'd use the ~ on the outer fence and backticks on the inner fence?